styles/App.scss

```diff
@@ -0,0 +1,22 @@
+$border-radius: 15px;
+$box-shadow-color: $base-orange;
+$box-shadow-simple: 0px 4px 4px rgba(0, 0, 0, 0.25);
+$box-shadow-property: var($box-shadow-simple), 4px 4px 40px $base-orange,
+    inset 4px 5px 4px rgba(0, 0, 0, 0.5);
+$base-black: #000000;
+$base-dark-blue: #14213d;
+$base-light-orange: #ffb703;
+$base-orange: #fca311;
```

Comment:
Te kolory to chyba wyżej musimy zadeklarować bo na razie to się Sass nie buduje :)

Reply:
done